### PR TITLE
DAT-2760 Support images in diffent formats

### PR DESCRIPTION
**Pull request by sebastianmarzjan**
- replace variables in image tag
- remove obsolete test

### CODE REVIEW

File: extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php [hunk 1]
```diff
@@ -65,12 +65,32 @@ protected function getValueWithDefault( \SimpleXMLElement $xmlNode ) {
...
 		return null;
 	}
 
+	protected function getRawInfoboxData ( $key ) {
+		$data = isset( $this->infoboxData[ $key ] ) ? $this->infoboxData[ $key ] : null;
+		return $data;
+	}
+
 	protected function getInfoboxData( $key ) {
 		$data = isset( $this->infoboxData[ $key ] ) ? $this->infoboxData[ $key ] : null;
```
Comment (idradm) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php]: Can't you invoke getRawInfoboxData here?

File: extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php [hunk 2]
```diff
@@ -65,12 +65,32 @@ protected function getValueWithDefault( \SimpleXMLElement $xmlNode ) {
 		return $value;
 	}
 
+	protected function getRawValueWithDefault( \SimpleXMLElement $xmlNode ) {
+		$source = $this->getXmlAttribute( $xmlNode, self::DATA_SRC_ATTR_NAME );
+		$value = null;
+		if ( !empty( $source ) ) {
+			$value = $this->getInfoboxData( $source );
+		}
```
Comment (idradm) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php]: Those few lines are duplicated in both getValueWithDefault and getRawValueWithDefault methods. Can we extract it?

File: extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php [hunk 3]
```diff
@@ -65,12 +65,32 @@ protected function getValueWithDefault( \SimpleXMLElement $xmlNode ) {
...
+		return $value;
+	}
+
 	protected function getXmlAttribute( \SimpleXMLElement $xmlNode, $attribute ) {
 		if ( isset( $xmlNode[ $attribute ] ) )
 			return (string)$xmlNode[ $attribute ];
 		return null;
 	}
 
+	protected function getRawInfoboxData ( $key ) {
```
Comment (idradm) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php]: Seems like this method is not used at all, is that ok?

**Comment by idradm** [comment]
few issues, additionally few more tests for base Node class could be awesome

**Comment by idradm** [comment]
LG

**Comment by jacekjursza** [comment]
LG

**Pull request merged**